src/pipelines/hospitalizations/config.yaml

```diff
@@ -113,6 +113,8 @@ sources:
     fetch:
       - url: "https://www.gov.scot/binaries/content/documents/govscot/publications/statistics/2020/04/coronavirus-covid-19-trends-in-daily-data/documents/covid-19-data-by-nhs-board/covid-19-data-by-nhs-board/govscot%3Adocument/COVID-19%2Bdata%2Bby%2BNHS%2BBoard.xlsx"
     test:
+      # TODO(owahltinez): Skip because the data source changed format
```

Comment:
I thought I fixed this data source - did the format change again?

Reply:
Unfortunately, tests started failing again. I was planning on taking a look after submitting this CL.

Reply:
Dang, fair enough.